### DISCUSSION TITLE
How do you in UK pronounce EPOCH ...

### DISCUSSION THREAD

**Discussion by DanRRRR** [epochpic/epoch#488] [General]
1) Epok, eh-puhk  or simply Epik, the last one because sounds like PIC ??

2) How correctly, acceptable and incorrectly reference it: Epoch, EPOCH, epoch ?

3) Also, do (Epoch) models the Kerr effect which affects self-focusing?

4) Do you plan to add electron collisions with the neutral atoms ?

**Comment by Status-Mirror** [comment]
- EE-pok. See this video: https://youtu.be/6qUASw244FY - timestamp 37s
- I'd advise EPOCH, as it's an acronym (Extendable PIC Open-source Collaboration, silent H)
- EPOCH should be able to model the refractive index for plasma (as this is a consequence of Maxwell's equations), but it won't be able to do so for other states of matter
- No current plans for Warwick to develop an electron/neutral collisions package (other than collisional ionisation, which is already present). EPOCH source-code is freely available, so other groups could add this if they wish, and we have added user-written physics packages to the main code before.
